geiger/src/graph/Graph.js

```diff
@@ -62,6 +69,10 @@ export default class Graph extends Events {
     }
   }
 
+  /**
+   * Add a node to the graph.
+   * @param {!Node} node
```

Comment:
Where is `Node` defined? (as in `@typedef`)

Reply:
Since `Node` is a module already imported via `import ...`, I simply ignore the `@typedef`.